JakinChan200/site
# Scrolling site

## Overview
 * Updated favicon name
 * Filled Experience description
 * Updated Resume icon

### To Do
 * N/A

## Materials
- N/A


## Pictures


## Manual Testing Guide
- Navigate to `site`
- Run `npm i`
- Run `npm run dev`

Pull requests should receive approval before merge.
